SQA_AT/src/test/java/api/StoreTest.java

```diff
@@ -0,0 +1,52 @@
+package api;
+
+import client.StoreClient;
+import io.restassured.response.Response;
+import model.Order;
+import org.testng.annotations.Test;
+
+import java.net.HttpURLConnection;
+
+import static org.testng.Assert.assertEquals;
+
+public class StoreTest {
+    @Test
+    public void getStoreInventory() {
+        StoreClient storeClient = new StoreClient();
```

Comment:
pls define general instances of the classes on a class level (as instance variable) and do not use it on a method level (as local variable)